database/postgresql/postgresql_user.py

```diff
@@ -113,6 +113,12 @@
     required: false
     default: null
     version_added: '1.4'
+  no_password_changes:
+    description:
+      - if C(yes), don't inspect database for password changes. Effective when C(pg_authid) is not accessible (such as AWS RDS). Otherwise, make password changes as necessary.
+    required: false
+    default: 'yes'
```

Comment:
default is incorrectly stated as 'yes', code has 'no' which keeps backwards compatibility